VERSION

```diff
@@ -1 +1 @@
-1.1.1
+1.1.2
```

Comment:
This seems odd - what branch are you working off of?

Reply:
Actually this is my mistake I didn't bump to 2.0.0! The version file should have that.